### DISCUSSION TITLE
WSS Capable?

### DISCUSSION THREAD

**Discussion by lblast** [doghappy/socket.io-client-csharp#273] [Q&A] [answered]
Hello, 
I'm pretty new to websockets in general, and I couldn't find anything in the documentation. Does this client support both WS and WSS? Is there a way to configure this or is the behavior default? Does it simply depend on the server URL that I pass when creating SocketIO?

Thanks!

**Comment by doghappy** [comment, marked as answer]
depends on your url

**Comment by lblast** [comment]
I see so it just matters if my server URL is http or https?

**Reply by doghappy** [reply]
exactly
